bootloader/bootloader_test.go

```diff
@@ -96,21 +95,31 @@ func (s *bootenvTestSuite) TestInstallBootloaderConfigNoConfig(c *C) {
 
 func (s *bootenvTestSuite) TestInstallBootloaderConfig(c *C) {
 	for _, t := range []struct {
-		gadgetFile, systemFile string
-		opts                   *bootloader.Options
+		name           string
+		gFile, sysFile string
```

Comment:
gFile is a bit too much of a contraction here I think,  g made me think of grub, not gadget for a second